magenta/protobuf/generator.proto

```diff
@@ -65,10 +65,15 @@ message GenerateSequenceResponse {
 
 // Bundle for wrapping a generator id, checkpoint file, and metagraph.
 // Intended to make sharing pre-trained models easier.
+// Next ID: 5
 message GeneratorBundle {
   // Details about the generator that created this bundle.
   GeneratorDetails generator_details = 1;
 
+  // A short text description of the bundle (e.g., training data, hparams,
+  // etc.).
+  string description = 4;
```

Comment:
I think i'd prefer we make a sub-message for bundle-specific metadata. I could see wanting to add other fields (e.g., author) at some point.